krummstab/__init__.py

```diff
@@ -22,6 +22,8 @@ class    - set of teams that are in the same exercise class
 import textwrap
 from importlib import resources
 import jsonschema
+import xlsxwriter
```

Comment:
I'd move most of the logic into a new file (`summaries.py`?) to declutter the init file.